WordPress/gutenberg
# [RNMobile] iOS side of bridge for media collection blocks

`gutenberg-mobile` PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2611



## Description


This PR implements the native iOS side for media collection blocks.

Some of these new methods can not be tested until the iOS side of a media collection block is implemented.

## How has this been tested?




- Run metro project from the [`gutenberg-mobile` branch](https://github.com/wordpress-mobile/gutenberg-mobile/pull/2611).
- Run the iOS demo app using Xcode.
- Add a new `Story` block.
  - Check the Xcode console that it prints: `gutenbergDidRequestMediaSaveSync()`
- Select the new block and press on `ADD MEDIA`.
  - Check the Xcode console that prints: `gutenbergDidRequestMediaFilesEditorLoad(_:blockId:)`
- Smoke test Image and Video blocks (adding media from device)


## Checklist:
